gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,9 @@
-#Wed Jan 21 09:37:07 CET 2015
+#Tue Jan 27 09:03:37 BRST 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+>>>>>>>=ad51598864e0ae9d9bc28238f2c4d94e65465286
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.2.1-all.zip
+=\=\=\=\=\=\=
+<<<<<<<=HEAD
```

Comment:
No problem, but please, send me an email instead of use G+ or twitter. I hate G+. You have my email in the README.md